src/NuGet.Clients/NuGet.PackageManagement.UI/UserInterfaceService/INuGetUI.cs

```diff
@@ -83,6 +83,10 @@ public interface INuGetUI
         /// </summary>
         PackageIdentity SelectedPackage { get; }
 
+        int SelectedIndex { get; }
+        int NumRecommended { get; }
+        bool RecommendPackages { get; }
```

Comment:
Logically, if feels weird that these are defined in an interface.

Reply:
Similar feedback as above about the spacing and naming.

Reply:
Removed these from the interface. But they are still in NuGetUI.cs. Should I try to find a way to remove them there as well?